src/styles/main.scss

```diff
@@ -0,0 +1,65 @@
+$day-size: 100px;
+$font-size: 30px;
+$gap: 1px;
+$calendar: ('mon' 0, 'tue' 1, 'wed' 2, 'thu' 3, 'fri' 4, 'sat' 5, 'sun' 6);
+
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  padding: 10px;
+  gap: 1px;
+  width: 7 * $day-size + 6 * $gap;
```

Comment:
The calculation for the width uses `6 * $gap`, which is hardcoded. Consider using a variable for `6` to improve maintainability and flexibility.